src/scripts/Player.ts

```diff
@@ -18,6 +18,8 @@ class Player {
     private _shinyList: boolean[];
     private _notCaughtBallSelection: GameConstants.Pokeball;
     private _alreadyCaughtBallSelection: GameConstants.Pokeball;
+    private _sortOption: KnockoutObservable<string>;
```

Comment:
Perhaps create an array from the enumerator options:

https://stackoverflow.com/questions/18150659/converting-enums-to-array-of-values-putting-all-json-values-in-an-array

https://github.com/Microsoft/TypeScript/issues/12877